### PR TITLE
Add recipe for pycovjson

### PR DESCRIPTION
**Pull request by lewismc**
@RileyWilliams @letmaik

### CODE REVIEW

File: recipes/pycovjson/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+test:
+  imports:
+    - pycovjson
+
+about:
+  home: https://github.com/Reading-eScience-Centre/pycovjson
+  license: BSD 3-clause
+  license_family: BSD
+  license_file: LICENSE
```
Comment (synapticarbors) [inline recipes/pycovjson/meta.yaml]: It looks like the license file is not distributed with the source. Please raise and issue upstream and copy the link to the issue as a comment in the recipe. Then you can manually include it:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

File: recipes/pycovjson/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```
Comment (synapticarbors) [inline recipes/pycovjson/meta.yaml]: It looks like this package has entry points. Please include them:

https://github.com/Reading-eScience-Centre/pycovjson/blob/master/setup.py#L33
https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

File: recipes/pycovjson/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    - python
+    - setuptools
+  run:
+    - python
+    - xarray
+    - numpy
+    - pandas
+    - pymongo
+
+test:
```
Comment (synapticarbors) [inline recipes/pycovjson/meta.yaml]: Please test the command line tools:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands

File: recipes/pycovjson/meta.yaml (hunk 2 above)
Comment (synapticarbors) [inline recipes/pycovjson/meta.yaml]: This package looks like a candidate for `noarch: python`

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

File: recipes/pycovjson/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (synapticarbors) [inline recipes/pycovjson/meta.yaml]: Is this package only for python 3? If so please add:

```
build:
   ...
   skip: True  # [py2k]
```

File: recipes/pycovjson/meta.yaml (hunk 2 above)
Comment (lewismc) [reply recipes/pycovjson/meta.yaml]: ack, done

Comment (lewismc) [reply recipes/pycovjson/meta.yaml]: ack, done

File: recipes/pycovjson/meta.yaml (hunk 4 above)
Comment (lewismc) [reply recipes/pycovjson/meta.yaml]: ack, done

File: recipes/pycovjson/meta.yaml (hunk 3 above)
Comment (lewismc) [reply recipes/pycovjson/meta.yaml]: ack, done

File: recipes/pycovjson/meta.yaml (hunk 1 above)
Comment (lewismc) [reply recipes/pycovjson/meta.yaml]: ack, done

ISSUE: https://github.com/Reading-eScience-Centre/pycovjson/issues/38
PULL REQUEST: https://github.com/Reading-eScience-Centre/pycovjson/pull/39

**Comment by lewismc** [comment]
Thank you very much @synapticarbors I really appreciator he review. Learning new things about condo-forge every time I submit a new recipe.

File: recipes/pycovjson/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+test:
+  imports:
+    - pycovjson
+  commands:
+    - pycovjson-convert -h
+    - pycovjson-viewer -h
+
+about:
+  about:
+    license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (synapticarbors) [inline recipes/pycovjson/meta.yaml]: It looks like there is a possible indentation issue here, and there is another `license_file` entry below still.

**Comment by lewismc** [comment]
Thank you @synapticarbors

**Pull request merged**